rewrite-java-21/src/main/java/org/openrewrite/java/isolated/ReloadableJava21TypeMapping.java

```diff
@@ -47,8 +48,8 @@ class ReloadableJava21TypeMapping implements JavaTypeMapping<Tree> {
     private final JavaTypeCache typeCache;
 
     public JavaType type(com.sun.tools.javac.code.@Nullable Type type) {
-        if (type == null || type instanceof Type.ErrorType || type instanceof Type.PackageType || type instanceof Type.UnknownType ||
-                type instanceof NullType) {
+        if (type == null || type instanceof Type.ErrorType || type instanceof Type.PackageType || matchesUnknownType(type)
+                || type instanceof NullType){
```

Comment:
```suggestion
        if (type == null || type instanceof Type.ErrorType || type instanceof Type.PackageType || matchesUnknownType(type) ||
                type instanceof NullType) {
```

Reply:
This is indeed a convention we maintain on the project; normally there's a bot to point these out on the other projects.